rally-domain/src/main/groovy/yakworks/rally/mail/services/MailMessageSender.groovy

```diff
@@ -33,25 +36,31 @@ class MailMessageSender {
      * calls mailgunMessagesApi.sendMessage using the MailgunConfig.defaultDomain
      *
      */
-    Result send(MailMessage mailMessage){
-        MailTo mailTo
+    Result send(MailMessage mailMessage) {
         Result result
+
         try {
-            mailTo = convertMailMessage(mailMessage)
-            // mailService.send should never throw ex and should return result
+            MailTo mailTo = convertMailMessage(mailMessage)
+            //validation throws a problem exception if validation fails
+            validateEmail(mailTo.to[0])
```

Comment:
why are you only validating the first one?

Reply:
`convertMailMessage` puts just single value in [mailTo](https://github.com/yakworks/gorm-tools/blob/a0ce39d0218e78d28f4353b66bb968ac94c3bc97/rally-domain/src/main/groovy/yakworks/rally/mail/services/MailMessageSender.groovy#L89)